composer.json

```diff
@@ -28,7 +28,7 @@
         "laravel/framework": "^8.0",
         "livewire/livewire": "^2.7",
         "lychee-org/nestedset": "^6",
-        "lychee-org/php-exif": "^0.7.4",
+        "lychee-org/php-exif": "^0.7.5",
```

Comment:
I wonder if you should agree upon some kind of "dependency update strategy". For example, in this case I personally would not have updated `composer.json`, but only run `composer update` and generated a new `composer.lock`. But that is mostly matter of taste.